From efadbf53a80421e27079de11384357f23257e5fa Mon Sep 17 00:00:00 2001
From: Erich Gubler <erichdongubler@gmail.com>
Date: Thu, 19 Jan 2023 00:20:06 -0500
Subject: [PATCH] build(linux): disable `PoolAlloc` to work around `asan`
 failures

We noticed this causing problems in [Firefox CI][failing-ci].

[failing-ci]: https://treeherder.mozilla.org/jobs?repo=autoland&author=egubler%40mozilla.com&selectedTaskRun=e6KdZgnNTCOE9weaOnTjVQ.0
---
 BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/BUILD.gn b/BUILD.gn
index 81d5cb41c1..d4e9a41285 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -578,7 +578,7 @@ template("angle_common_lib") {
       ":includes",
     ]
     public_configs += [ ":angle_common_config" ]
-    if (angle_has_build && use_fuzzing_engine) {
+    if (angle_has_build) {
       all_dependent_configs = [ ":angle_disable_pool_alloc" ]
     }
 
-- 
2.55.0

